package.json

```diff
@@ -26,7 +26,6 @@
     "@favware/npm-deprecate": "^1.0.5",
     "@favware/rollup-type-bundler": "^1.0.10",
     "@joshdb/core": "3.0.0-next.c67d6d9.0",
-    "@joshdb/provider": "^1.1.0-next",
```

Comment:
Turbo build was using this version over the version in each package, which might be another issue. As to why I removed it instead of updating it, I didn't know we used it for benchmarks

Reply:
Yes that is fine.